src/containers/workschedule/AppointmentContainer.js

```diff
@@ -169,8 +170,8 @@ const AppointmentContainer = ({
   const assignedWorksites :List = worksitesByWorksitePlan.valueSeq().toList();
 
   let checkedInSymbol = CheckedInIcon;
-  const checkInHoursEqualAppointmentHours = doesCheckInMatchAppointment(numHours, checkIn);
-  if (!checkInHoursEqualAppointmentHours) {
+  const numberHoursWorked = getPropertyValue(checkIn, [HOURS_WORKED, 0]);
+  if (!numberHoursWorked) {
```

Comment:
No change required, just a thought. If I were doing this, I'd do `numberHoursWorked === 0`. I think it's a little easier to read vs the `!` and you're strictly checking for the number vs a potentially falsy value.

Reply:
okay ya, i agree—i'll update!